ws-dotnet/files/workspace/README.md

```diff
@@ -0,0 +1,57 @@
+     ,-----.,--.                  ,--. ,---.   ,--.,------.  ,------.
+    '  .--./|  | ,---. ,--.,--. ,-|  || o   \  |  ||  .-.  \ |  .---'
+    |  |    |  || .-. ||  ||  |' .-. |`..'  |  |  ||  |  \  :|  `--,
+    '  '--'\|  |' '-' ''  ''  '\ `-' | .'  /   |  ||  '--'  /|  `---.
+     `-----'`--' `---'  `----'  `---'  `--'    `--'`-------' `------'
+    -----------------------------------------------------------------
+
+
+Hi there! Welcome to Cloud9 IDE! This is the template for .NET Core!
+
+You'll find 4 sample applications within this image
+- ConsoleApplicationCS (CSharp)
+- ConsoleApplicationFS (FSharp)
+- WebApp
+
+
+All applications have their own runners and can be executed from the top "Run" menu by right-clicking on it
```

Comment:
Where are these runners? Currently we don't have the ones in the IDE in a publicly contributable repository, but if you can provide them to me I can get them added.